roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -24,3 +24,32 @@
     disable_gpg_check: true
     state: present
   when: foreman_repositories_katello_version is defined
+
+- when:
+    - foreman_repositories_katello_version is defined
+    - ansible_distribution_major_version == '8'
+  block:
+    - name: Install dnf-config-manager
+      package:
+        name: dnf-plugins-core
+        state: present
+
+    - name: Get pki-core module info
+      command: dnf module list --enabled
+      register: enabled_modules
+      changed_when: false
+
+    - name: Enable pki-core module
+      command: dnf module enable pki-core -y
+      when: "'pki-core' not in enabled_modules.stdout"
```

Comment:
doesn't the installer do that for us these days?

Reply:
I thought it should but it fails without this.

Reply:
weird, but nothing we have to fix today.